### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by petrparolek**
- upgrade PHPStan
- unify CS
- added schedule jobs in GitHub Actions

### CODE REVIEW

File: tests/cases/AbstractMenuItemsContainerTest.php [hunk 1]
```diff
@@ -26,7 +24,11 @@ public function testItems(): void
 		});
 
 		$container = $this->createPartialMockAbstractMenuItemsContainer(null, [
-			$menu, $linkGenerator, $translator, $authorizator, $itemFactory,
+			$menu,
+		$linkGenerator,
```
Comment (foxycode) [inline tests/cases/AbstractMenuItemsContainerTest.php:28]: This looks like bad indentation

File: tests/cases/AbstractMenuItemsContainerTest.php [hunk 2]
```diff
@@ -107,7 +109,11 @@ public function testFindActiveItem(): void
 
 		$container = $this->createPartialMockAbstractMenuItemsContainer(function (
 			MockInterface $container
-		) use ($itemA, $itemB, $itemC): void {
+		) use (
+			$itemA,
+			$itemB,
+			$itemC
+): void {
```
Comment (foxycode) [inline tests/cases/AbstractMenuItemsContainerTest.php:116]: Also bad indentation

File: tests/cases/AbstractMenuItemsContainerTest.php [hunk 3]
```diff
@@ -140,7 +146,10 @@ public function testVisibleItemsOn(): void
 
 		$container = $this->createPartialMockAbstractMenuItemsContainer(function (
 			MockInterface $container
-		) use ($itemA, $itemB): void {
+		) use (
+			$itemA,
+			$itemB
+): void {
```
Comment (foxycode) [inline tests/cases/AbstractMenuItemsContainerTest.php:152]: Also bad indentation

**Review by foxycode — COMMENTED** [review]
Great work :)

File: tests/cases/AbstractMenuItemsContainerTest.php (hunk 1 above)
Comment (petrparolek) [reply tests/cases/AbstractMenuItemsContainerTest.php:28]: I don't know why phpcsfix did that. :open_mouth:

File: tests/cases/AbstractMenuItemsContainerTest.php (hunk 3 above)
Comment (petrparolek) [reply tests/cases/AbstractMenuItemsContainerTest.php:152]: I don't know why phpcsfix did that. :open_mouth:

**Pull request merged**